ansible/roles/products/teleop/server/desktop-icons/tasks/main.yml

```diff
@@ -12,32 +12,105 @@
     state: directory
   become: yes
 
-- name: Copy Teleop_Control_Machine_Launch desktop icon
+#NUC demohand A 
```

Comment:
Maybe it will make sense to have the same functionality in separate tasks file and execute it 3 times with different parameters. Please ask @ToivoHn for more details.

Reply:
I have created a (hopefully) better structure